velox/functions/prestosql/window/WindowFunctionsRegistration.cpp

```diff
@@ -15,11 +15,10 @@
  */
 #include "velox/functions/prestosql/window/WindowFunctionsRegistration.h"
 #include "velox/exec/Aggregate.h"
+#include "velox/exec/AggregateWindow.h"
```

Comment:
Do we need this include? Also, we may not need Aggregate.h include. Please, double check.